golang/outbound.go

```diff
@@ -87,6 +87,10 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string, callOpti
 	call.createStatsTags(c.commonStatsTags)
 	call.log = c.log.WithFields(LogField{"Out-Call", requestID})
 
+	if callOptions.Format != HTTP {
+		call.commonStatsTags["target-endpoint"] = string(operation)
```

Comment:
can you pass the `callOptions` and `operation` into createStatsTags and move this commonStatsTag setter into that method